beacon-chain/core/helpers/validators.go

```diff
@@ -675,6 +675,30 @@ func ValidatorMaxEffectiveBalance(val *ethpb.Validator) uint64 {
 	return params.BeaconConfig().MinActivationBalance
 }
 
+// SwitchToCompoundingValidator used to queue excess active balance based on validator.
+//
+// Spec definition:
+//
+//	 def switch_to_compounding_validator(state: BeaconState, index: ValidatorIndex) -> None:
+//		validator = state.validators[index]
+//		if has_eth1_withdrawal_credential(validator):
+//		    validator.withdrawal_credentials = COMPOUNDING_WITHDRAWAL_PREFIX + validator.withdrawal_credentials[1:]
+//		    queue_excess_active_balance(state, index)
+func SwitchToCompoundingValidator(s state.BeaconState, idx primitives.ValidatorIndex) error {
```

Comment:
This already exists in develop. Please delete this and use `github.com/prysmaticlabs/prysm/v5/beacon-chain/core/electra.SwitchToCompoundingValidator`

Reply:
I left this here due to the cyclic import due to process deposits.

Reply:
There must be a better way than duplicating a method due to cyclic imports